ansible/deploy.yml

```diff
@@ -64,6 +61,26 @@
             state: present
           become: true
 
+    - name: Check if uv is already installed
+      ansible.builtin.stat:
+        path: "/home/deploy/.cargo/bin/uv"
+      register: uv_installed
+      when: install_system_dependencies is true
+      tags: [uv-install]
+
+    - name: Download uv install script
```

Comment:
Similarly, `ansible.builtin.get_url` by default will only run if `/tmp/uv.install.sh` does not already exist by default.

So in this case, I think we can likely get way with not needing the `and not uv_installed` check either, because we only really want to fetch the uv install script if `"/tmp/uv.install.sh"` doesn't already exist.